### PR TITLE
adding Mikado file

### PR DESCRIPTION
**Pull request by xuan73**

### CODE REVIEW

**Review by uncleye — COMMENTED** [review]
shenme jb wanyi

File: Mikado.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mikado
```
Comment (uncleye) [inline Mikado.txt]: 这特么牛逼

**Review by uncleye — CHANGES_REQUESTED** [review]
改改吧

File: Mikado.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-mikado
+Mikado is a method of plan what to do and how to do a project.
+Let's talk about it later.
```
Comment (xuan73) [inline Mikado.txt:2]: Let's talk about it later.

**Pull request merged**